cmd/hypercc/main.go

```diff
@@ -17,27 +17,28 @@ limitations under the License.
 package main
 
 import (
-	"fmt"
 	"os"
 	"path"
 
+	"github.com/spf13/cobra"
+
 	capp "github.com/kubernetes-incubator/cluster-capacity/cmd/cluster-capacity/app"
 	gapp "github.com/kubernetes-incubator/cluster-capacity/cmd/genpod/app"
 )
 
 func main() {
 	switch path.Base(os.Args[0]) {
 	case "cluster-capacity":
-		cmd := capp.NewClusterCapacityCommand()
-		if err := cmd.Execute(); err != nil {
-			fmt.Println(err)
-			os.Exit(1)
-		}
+		cmdExecute(capp.NewClusterCapacityCommand())
 	case "genpod":
-		cmd := gapp.NewGenPodCommand()
-		if err := cmd.Execute(); err != nil {
-			fmt.Println(err)
-			os.Exit(1)
-		}
+		cmdExecute(gapp.NewGenPodCommand())
+	default:
+		cmdExecute(NewHyperCCCommand())
+	}
+}
+
+func cmdExecute(cmd *cobra.Command) {
+	if err := cmd.Execute(); err != nil {
+		os.Exit(1)
```

Comment:
Actually when an error is being printed in github.com/spf13 package, then it is being duplicated, otherwise not. so that can not be avoided I think.

Anyway I have updated the code to include Println in the main.go so the code is as you suggested.